src/test/kotlin/me/serce/solidity/ide/typing/SolBlockCommentEnterHandlerTest.kt

```diff
@@ -53,6 +64,17 @@ class SolBlockCommentEnterHandlerTest : SolTestBase() {
     myFixture.type('\n')
   }
 
+  fun testBlockCommentContinuationFromSingleLine() = checkByText(
+    "/*<caret>*/",
```

Comment:
@Paulalex85 I think this is the `/*|*/` you mentioned? Unless I misunderstood it?